docs/installation/postgresql.md

```diff
@@ -40,6 +40,9 @@ connection string.
 
 ```yml
 DATABASE_URL: postgresql://USERNAME:PASSWORD@localhost
+
+# Incase you use a separate database for test. This is optional.
```

Comment:
This might be a little confusing, we do use a separate database but sometimes that database might be at a different URL. Might be more clear to reword this a little and are URL to it.

Reply:
Good point. Updated!